docs/source/dev_guide/testing_e3sm_unified.rst

```diff
@@ -0,0 +1,361 @@
+.. _testing_e3sm_unified:
+
+*************************************
+Testing zppy for E3SM Unified release
+*************************************
+
+This tutorial covers testing ``zppy`` for an E3SM Unified release.
+Examples are from testing for E3SM Unified v1.5.0.
+
+Parameters
+==========
+
+Only a few parameters in the configuration file need to change for different machines.
+Be sure to change the file paths to fit your own workspaces.
+
+Chrysalis:
+
+- <INPUT>: /lcrc/group/e3sm/ac.forsyth2/E3SM_simulations/e3sm_unified_test_simulation/20210528.v2rc3e.piControl.ne30pg2_EC30to60E2r2.chrysalis
+- <OUTPUT>: /lcrc/group/e3sm/ac.forsyth2/E3SM_simulations/e3sm_unified_test_simulation/
+- <WWW>: /lcrc/group/e3sm/public_html/diagnostic_output/ac.forsyth2/E3SM/e3sm_unified_test_simulation/
+- <PARTITION>: compute
+- <ENVIRONMENT_COMMANDS>: source /lcrc/soft/climate/e3sm-unified/test_e3sm_unified_1.5.0rc8_chrysalis.sh
+- <MAPPING_FILE>: /home/ac.zender/data/maps/map_ne30pg2_to_cmip6_180x360_aave.20200201.nc
+- <REFERENCE_DATA_PATH>: /lcrc/soft/climate/e3sm_diags_data/obs_for_e3sm_diags/climatology
+- <OBS_TS>: /lcrc/soft/climate/e3sm_diags_data/obs_for_e3sm_diags/time-series
+- <DC_OBS_CLIMO>: /lcrc/group/e3sm/public_html/e3sm_diags_test_data/unit_test_complete_run/obs/climatology
+
+Compy:
+
+- <INPUT>: /compyfs/fors729/e3sm_unified_test_zppy/20210528.v2rc3e.piControl.ne30pg2_EC30to60E2r2.chrysalis
+- <OUTPUT>: /compyfs/fors729/e3sm_unified_test_zppy/
+- <WWW>: /compyfs/www/fors729/E3SM/e3sm_unified_test_simulation/
+- <PARTITION>: slurm
+- <ENVIRONMENT_COMMANDS>: source /share/apps/E3SM/conda_envs/test_e3sm_unified_1.5.0rc8_compy.sh
+- <MAPPING_FILE>: /compyfs/zender/maps/map_ne30pg2_to_cmip6_180x360_aave.20200201.nc
+- <REFERENCE_DATA_PATH>: /compyfs/e3sm_diags_data/obs_for_e3sm_diags/climatology
+- <OBS_TS>: /compyfs/e3sm_diags_data/obs_for_e3sm_diags/time-series
+- <DC_OBS_CLIMO>: /compyfs/e3sm_diags_data/obs_for_e3sm_diags/climatology
+
+Cori:
+
+- <INPUT>: /global/cscratch1/sd/forsyth/e3sm_unified_test_zppy/20210528.v2rc3e.piControl.ne30pg2_EC30to60E2r2.chrysalis
```

Comment:
The input data on Cori are  saved on scratch system, which has a purge policy (12 weeks). To be safe, maybe we should move them to somewhere more permanently, i.e. under cfs file system?

Reply:
Do we anticipate keeping this data around? It seems like it's only useful for testing -- i.e., we could just delete it after E3SM Unified is released.

Furthermore, doesn't `cfs` have a significantly smaller space quota than `cscratch`?

Reply:
I think this dataset is useful for testing diagnostics tools on v2 data. It's helpful to keep it. What's the size of the directory?

Reply:
Ah ok that makes sense. `/global/cscratch1/sd/forsyth/e3sm_unified_test_zppy/20210528.v2rc3e.piControl.ne30pg2_EC30to60E2r2.chrysalis` is 2 terabytes in size.

Reply:
hmm, it's bigger than i thought. I found that cfs  and scratch have the same space quota (20T).  but I guess either way is fine. We can always transfer if the data are being purged.

Reply:
I can add a note in the documentation for testers to think about what file path is best to store their data.